docs/development/extensions-core/kafka-supervisor-reference.md

```diff
@@ -144,14 +144,15 @@ Multiple topics can be passed as a regex pattern as the value for `topicPattern`
 ingest data from clicks and impressions, you will set `topicPattern` to `clicks|impressions` in the IO config.
 Similarly, you can use `metrics-.*` as the value for `topicPattern` if you want to ingest from all the topics that
 start with `metrics-`. If new topics are added to the cluster that match the regex, Druid will automatically start
-ingesting from those new topics. If you enable multi-topic ingestion for a datasource, downgrading to a version
-lesser than 28.0.0 will cause the ingestion for that datasource to fail.
-
-When ingesting data from multiple topics, the partitions are assigned based on the hashcode of topic and the id of the 
-partition within that topic. The partition assignment might not be uniform across all the tasks. It's also assumed 
-that partitions across individual topics have similar load. It is recommended that you have a higher number of 
-partitions for a high load topic and a lower number of partitions for a low load topic. Assuming that you want to 
-ingest from both high and low load topic in the same supervisor. 
+ingesting from those new topics. A topic name that only matches partially such as `my-metrics-12` will not be
+included for ingestion. If you enable multi-topic ingestion for a datasource, downgrading to a version lesser than
+28.0.0 will cause the ingestion for that datasource to fail.
+
+When ingesting data from multiple topics, the partitions are assigned based on the hashcode of the topic name and the
```

Comment:
This whole part could be under a sub-heading of partition assignment. And the statements should be rephrased/restructured for clarity, maybe as a list?

Reply:
I am not sure. Personally I prefer the current format itself since there is not a lot of content.